src/odoc/bin/main.ml

```diff
@@ -403,45 +403,46 @@ end = struct
   let targets = Targets.(cmd, info)
 end
 
-module Odoc_uri : sig
+let reference =
+  let doc = "The reference to be resolved and whose uri to be generated." in
+  Arg.(required & pos 0 (some string) None & info ~doc ~docv:"REF" [])
+
+module Odoc_html_uri : sig
   val cmd : unit Term.t
 
   val info : Term.info
 end = struct
-  let prefix =
+  let root_dir =
```

Comment:
Just thinking, the prefix to append can also be something else that a directory (for instance a url to the online doc in v3).
So maybe a better name would be `root_uri`? :/

Reply:
How does `base` sound?

Reply:
Indeed. In this case adding a `/` at the end of it is wrong.